packages/connector/src/services/core/controllers/stream.ts

```diff
@@ -1,39 +1,46 @@
-import * as assert from 'assert'
-import { StreamServer } from '@interledger/stream-receiver'
 import { isIlpReply } from 'ilp-packet'
 import { RafikiContext, RafikiMiddleware } from '../rafiki'
 
-interface StreamControllerOptions {
-  serverSecret: Buffer
-  serverAddress: string
-}
+const CONNECTION_EXPIRY = 60 * 60 // seconds
```

Comment:
fwiw, this is configurable in the receipt-verifier
https://github.com/coilhq/receipt-verifier#receipt_ttl

Reply:
But the connector doesn't know what the receipt verifier has set for the ttl, right?

Reply:
Right.
I was confusing myself thinking ahead to rafiki also having an receipt verifier.
Not configuring the expiry and extending it for each new packet sounds good to me.